test/val/val_id_test.cpp

```diff
@@ -33,14 +35,9 @@ using std::ostringstream;
 using std::string;
 using std::vector;
 
-class ValidateID : public ::testing::Test {
- public:
-  virtual void TearDown() { spvBinaryDestroy(binary); }
-  spv_const_binary get_const_binary() { return spv_const_binary(binary); }
-  spv_binary binary;
-};
+using ValidateIdWithMessage = spvtest::ValidateBase<bool>;
 
-const char kGLSL450MemoryModel[] = R"(
+string kGLSL450MemoryModel = R"(
```

Comment:
FYI the Google C++ style guide forbids static-duration objects with C++ constructors.  std::string is an example.  But the reasons for it don't really apply for a test executable that starts and shuts down so trivially quickly.

https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables